src/lib/util/util.ts

```diff
@@ -711,6 +711,22 @@ export function isTextBasedChannel(channel: Channel) {
 	return channel.type === 'text' || channel.type === 'news';
 }
 
+/**
+ * Shuffles an array, returning it
+ * @param array The array to shuffle
+ */
+export const shuffle = <T>(array: readonly T[]): T[] => {
+	const clone = array.slice();
+	const shuffled = [] as T[];
+
+	for (const { } of array) {
+		const [value] = clone.splice(Math.floor(Math.random() * clone.length), 1);
+		shuffled.push(value);
+	}
+
+	return shuffled;
+};
```

Comment:
You should use this, it's more efficient, and once you do that, you can make HG use this utility in the next PR:

https://github.com/skyra-project/skyra/blob/475551c3b01b9b34cc655e01aa71b7a365a42e30/src/commands/Games/hungergames.ts#L167-L171

Reply:
Also unit test this utility.